### ISSUE TITLE
Training from a checkpoint on new data

### ISSUE THREAD

**Issue by shredpub** [OpenNMT/OpenNMT#195]
Hello, I want to fine tune a pretrained model on new data (incremental adaptation) using some new parameters (epochs, learning rate). However, I have been facing problems while using the train_from option due to varying vocabulary sizes (due to different data, hence different preprocessed .t7 files, hence different vocab) in both iterations. Could anyone please suggest a smooth way to achieve this?

**Comment by vince62s** [comment]
You can't do that. You need a common vocabulary for the 2 sets of data.

**Comment by shredpub** [comment]
Yes I know, hence I'm trying to find a way to train on new data but use the old vocabulary (I think it is generating the vocab automatically from the .t7 file)

**Comment by guillaumekln** [comment]
This is mentionned in the documentation : http://opennmt.net/OpenNMT/data/preparation/#vocabularies

When preprocessing your new data, you should reuse the generated vocabularies (`.dict` files) with the `-src_vocab` and `-tgt_vocab` options.

**Comment by shredpub** [comment]
Thank you, let me try that out. I had been looking for the same option in train.py.

**Comment by shredpub** [comment]
It does work. Thanks again. Closing the issue.
